### PR TITLE
ThanosPhobia The Disease no one asked for

### PR DESCRIPTION
**Pull request by Diegoflores31**
**What does this PR do:**
Thanos Has Arrived in NSS Cyberaid , he will bring Peace  , isulated gloves and Unlimited access for everyone** 

(everyone** that survives this stupid fuck disease)

**Images of sprite/map changes (IF APPLICABLE):**
If you did not make a map or sprite edit, you may leave this section blank. You may include a gif of your feature if you want

**Changelog:**
*Remove this line, and appropriate fields from the changelog*
:cl:
add: Thanosphobia Disease
/:cl:

### CODE REVIEW

**Comment by Quantum-M** [comment]
But why?

**Comment by Diegoflores31** [comment]
@Quantum-M  i just felt that we need some thanos memes but im not sure how to make someone become actual dust so i just made that lel

**Comment by AffectedArc07** [comment]
I have several complaints
- Meme PRs are shit as it is
- This one doesnt even have funny numbers
- You did all of this in the web editor 
- You didnt even add the file to the DME
- This is a shit meme
- You didnt use the template properly

File: code/datums/diseases/ThanosPhobia.dm [hunk 1]
```diff
@@ -0,0 +1,49 @@
+    
+/datum/disease/ThanosPhobia
+	form = "Disease"
+	name = "Infinity Disease"
+	max_stages = 5
+	spread_text = "Airborne"
+	cure_text = "Only a Major Force can Safe you From the Rage Of Thanos"
```
Comment (AffectedArc07) [inline code/datums/diseases/ThanosPhobia.dm]: Grammar

File: code/datums/diseases/ThanosPhobia.dm [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	cures = list("Uranium","ephedrine")
```
Comment (AffectedArc07) [inline code/datums/diseases/ThanosPhobia.dm]: Capitalisation

File: code/datums/diseases/ThanosPhobia.dm [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	agent = "Infinity Juice"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	cure_chance = 4// I trully dont know how do a 50% chance but welp i mean someone will read this and fix it later rigth ? i mean if it gets merged maybe
+	desc = "Thanos Killed half of the Galaxy , seems like you are the unlucky 50% so..... enjoy the ride ? "
```
Comment (AffectedArc07) [inline code/datums/diseases/ThanosPhobia.dm]: Spacing

File: code/datums/diseases/ThanosPhobia.dm [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+			if(prob(2))
+				to_chat(affected_mob, "<span class='danger'>your finguers starts becoming Dust!</span>")
+				affected_mob.adjustcloneLoss(50)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>You feel Thanos Dancing In your mind.</span>")
+				affected_mob.adjustOxyLoss(30)
+				affected_mob.emote("scream")
+		if(5)
+			if(prob(2))
+				to_chat(affected_mob, "<span class='userdanger'>You think you should paid attention to that Bwoink Sound.</span>")
```
Comment (AffectedArc07) [inline code/datums/diseases/ThanosPhobia.dm]: Meta

File: code/datums/diseases/ThanosPhobia.dm [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+			if(prob(2))
+				affected_mob.emote("cough")
+				to_chat(affected_mob, "<span class='danger'>You feel weird.</span>")
+			if(prob(2))
+				to_chat(affected_mob, "<span class='danger'>Your heart Speeds Up!</span>")
+			if(prob(5))
+				to_chat(affected_mob, "<span class='danger'>You feel a cold.</span>")
+		if(4)
+			if(prob(2))
+				to_chat(affected_mob, "<span class='userdanger'>I dont want to Go Mr.Captain</span>")
```
Comment (AffectedArc07) [inline code/datums/diseases/ThanosPhobia.dm]: Spacing

**Comment by CornMyCob** [comment]
Dusting someone would just be as easy as calling the dust proc but this gets a no from me, there's no need for this.

**Comment by Mitchs98** [comment]
This is honestly worse of a PR than the cancer disease that was made a while back by you, all things considered. The PR itself doesn't actually say what the disease does, the changelog isn't properly made, and the whole thing is littered with spelling and grammar issues. 

More diseases could be neat, but actual cancer and some random meme disease isn't one of them.

**Comment by Spacemanspark** [comment]
Why

**Comment by SkeletalElite** [comment]
I like the idea that this disease be non spreadable and making it admin mite
*snap

**Comment by Couls** [comment]
title is pretty accurate

**Comment by Fox-McCloud** [comment]
Meme PRs are great when done well.

This wasn't done well. It wasn't even 1% of done well.

**Pull request closed without merging**